pkg/manager/member/tidb_member_manager.go

```diff
@@ -402,7 +402,7 @@ func getTiDBConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 	if tc.IsHeterogeneous() {
 		tidbStartScriptModel.Path = controller.PDMemberName(tc.Spec.Cluster.Name) + ":2379"
```

Comment:
Do we need to update this one?

Reply:
No need, we just need to avoid trigger rolling update for the existing clusters.